docs/workflow/trimming/feature-switches.md

```diff
@@ -0,0 +1,56 @@
+# Libraries Feature Switches
+
+Starting with .NET 5 there are several [feature-switches](https://github.com/dotnet/designs/blob/master/accepted/2020/feature-switch.md) available which
+can be used to control the size of the final binary. They are available in all
+configurations but their defaults might vary as any SDK can set the defaults differently.
+
+## Available Feature Switches
+
+| MSBuild property name | C# property name | Description |
```

Comment:
I wouldn't call this `C# property name`. Instead I would call it `AppContext setting`. The AppContext switch is the public facing API here.

In most cases (all cases so far), the C# property name is not public and it doesn't need to match 1:1 with the AppContext string.